proposed/http-message-meta.md

```diff
@@ -202,14 +202,14 @@ times in a give request -- and which would require parsing the URI in order to
 determine (e.g., via `parse_url()`). Modeling URIs as value objects allows
 parsing once only, and simplifies access to individual segments. It also
 provides convenience in client applications by allowing users to create new
-instances of a base URI instance with just the segments that change (e.g.,
+instances of a base URI instance with only the segments that change (e.g.,
 updating the path only).
 
 ### Why does the request interface have methods for dealing with the request-target AND compose a URI?
 
 RFC 7230 details the request line as containing a "request-target". Of the four
 forms of request-target, only one is a URI compliant with RFC 3986; the most
-common form used, however, is origin-form, which represents the URI without the
+common form used, is origin-form, which represents the URI without the
```

Comment:
Same comment as before about removing `however,`; remove the preceding comma, too.